src/components/Main/Members/Members.tsx

```diff
@@ -37,6 +37,7 @@ export const Members = () => {
       cacheTime: 3000000,
       refetchOnWindowFocus: false, //these dont work properly eugh
       refetchOnMount: false,
+      enabled: !!churchId,
```

Comment:
I don't think this should be an issue, but turn churchId into a string first. if for some reason we have churchId as 0, !!0 == false. I don't see any world where this could happen or why it would happen, but for coding purposes, we should handle a case where churchId ===0

Reply:
so whats the right code? !!churchId.toString() ?

Reply:
lol... i dunno. that looks pretty lame tbh

Reply:
agh. i dunno actually, now that I think about it, "-1" would pass too

i guess just keep it lol

Reply:
Is churchId a number? If so, then:  
```suggestion
      enabled: churchId > 0,
```